core/services/ocr2/plugins/promwrapper/plugin.go

```diff
@@ -30,9 +35,21 @@ var (
 		float64(5 * time.Minute),
 		float64(10 * time.Minute),
 	}
-	labels = []string{"chainType", "chainID", "plugin", "oracleID", "configDigest"}
+	labels          = []string{"chainType", "chainID", "plugin", "oracleID", "configDigest", "epoch", "round"}
```

Comment:
`epoch` and `round` may result in too high of cardinality. As long as we can get time-series data, epoch and round might not be necessary.